src/components/Datathon/Leaderboard/index.tsx

```diff
@@ -0,0 +1,74 @@
+import { connectDB } from '@/db/config';
+import React from 'react';
+import Leaderboard, { ITeamData } from '@/db/models/leaderboard';
+
+export default async function LeaderboardTable() {
+  connectDB();
+
+  const leaderboards = await Leaderboard.find({})
+    .sort({ timestamp: -1 })
+    .limit(1)
+    .exec();
```

Comment:
We can improve the query by using `findOne()` instead of `find()` to avoid needing to deal with an array